charts/kubernetes-agent/templates/_helpers.tpl

```diff
@@ -80,13 +79,23 @@ The name of the secret to store the authentication information (bearer token/api
 {{- printf "%s-tentacle-server-auth" ( include "kubernetes-agent.name" . ) }}
 {{- end }}
 
+{{/*
+The name of the PersistentVolumeClaim to configure
+*/}}
+{{- define "kubernetes-agent.pvcName" -}}
+{{- if .Values.persistence.nfs.enabled }}
+{{- include "nfs.pvcName" . }}
+{{- else }}
+{{- printf "%s-pvc" (include "kubernetes-agent.fullName" .) }}
+{{- end }}
+{{- end }}
 
+{{/*
+This is passed to the Tentacle Container as JSON
+*/}}
 {{- define "kubernetes-agent.podVolumeYaml" -}}
-{{- if .Values.storage.nfs.enabled }}
 volumes:
 - name: tentacle-home
   persistentVolumeClaim:
-    claimName: {{ include "nfs.pvcName" .}}
-{{- end }}
-{{- end }}
-
+    claimName: {{ include "kubernetes-agent.pvcName" . }}
+{{- end }}
```

Comment:
In a future PR, this could just change to passing the `claimName` directly, rather than the JSON